static/js/lib/validation/profile.js

```diff
@@ -299,5 +299,10 @@ export const validateFinancialAid = (edit: FinancialAidState): FinancialAidValid
   if (!edit.checkBox) {
     errors['checkBox'] = 'You must agree to these terms';
   }
+
+  if (edit.income && /\D+/.test(edit.income)) {
+    errors['income'] = 'Please only use whole numbers.';
+  }
+
```

Comment:
could you add a quick test of this behavior in the `validation/profile_test.js` file?